### PR TITLE
Loading component

### PR DESCRIPTION
**Pull request by cammac60**
### What's this PR do? 

- Adds and styles the loading component. 
- Adds conditional rendering to display the loading component at the `/home` URL when projects haven't loaded. 

----------

### How should this be manually tested? 

- `git fetch`
- `git checkout loading-component`
- `npm start`
- Since this component only loads based on the status of the projects fetch it may be hard to manually test. I'd recommend 1.) Creating a new route in the app component with an exact path of `/loader` which renders the loader, then visiting that URL to check out styling. OR 2.) Purposely removing the projects fetch so that you can see the component from `/home`. Obviously, any changes made for testing purposes should be undone. 

----------

### Screenshots 

<details>

<summary>Loader 1</summary>

 

</details>
<details>

<summary>Loader 1</summary>

 

</details>

----------

### What tickets does this PR resolve? 

- #53

### CODE REVIEW

File: src/components/Loader/Loader.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  -webkit-animation: gradient-animation 10s ease infinite;
+  animation: gradient-animation 10s ease infinite;
+}
+
+@-webkit-keyframes gradient-animation {
+    0%{background-position:0% 50%}
+    50%{background-position:100% 50%}
+    100%{background-position:0% 50%}
+}
+@keyframes gradient-animation {
```
Comment (hfaerber) [inline src/components/Loader/Loader.scss:18]: This loader looks incredible!  I'm really impressed looking through the css for it!

File: src/App.js [hunk 2]
```diff
@@ -25,7 +26,7 @@ const App = () => {
   return (
     <div className="App">
       <Route exact path='/' render={() => <Login setUser={setUser}/>} />
-      <Route exact path='/home' render={() => <Home {...homeProps}/>} />
+      <Route exact path='/home' render={() => projects ? <Home {...homeProps}/> : <Loader />} />
```
Comment (hfaerber) [inline src/App.js]: I couldn't see the loader in action, even when commenting out the `setProjects` on line 18. 
We may need to change the conditional on line 29 to `projects.length ? <Home.......`

Before we merge this PR, Could you test this on your end?  Changing `projects` to `projects.length` on my end seemed to work.

**Review by hfaerber — CHANGES_REQUESTED** [review]
Can you please see my comment on App.js?

File: src/App.js (hunk 2 above)
Comment (cammac60) [reply src/App.js]: @hfaerber No problem, I added another commit. I was also able to verify that it's working on my end.

**Comment by hfaerber** [comment]
Thanks for making that adjustment!  Merging now.

**Pull request merged**